src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java

```diff
@@ -1654,6 +1654,8 @@ public RoleAction createRole()
     {
         checkPermission(Permission.MANAGE_EMOTES);
         Checks.notBlank(name, "Emote name");
+        Checks.notShorter(name, 2, "Emote name");
```

Comment:
Wouldn't it be more reasonable to use `Checks.check` here instead of inventing a new tool for two cases?

Or alternatively, have a `Checks.length` method that checks a min and max length (Maybe with -1 making them optional?)?

Reply:
Better would be `Checks.inRange(string, min, max, ...)` maybe but using just the raw check is less efficient and would require writing the same error string on each check, which is undesirable.